domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -64,14 +67,16 @@ await factory.SeedData(new List<TransferAgreement>()
                 SenderId = senderId,
                 SenderName = "nrgi A/S",
                 SenderTin = "44332211",
-                ReceiverTin = "12345678"
+                ReceiverTin = "12345678",
+                ReceiverReference = Guid.Parse("5b24891b-1ab7-48c0-b747-9341a7dc6eec")
```

Comment:
Why is this hardcoded? Could it just be `Guid.NewGuid()`?

If it not required to be hardcoded, then i would suggest changing it everywhere to `NewGuid`

Reply:
Changed to use Guid.NewGuid()